packages/frontend/amp/components/elements/Image.tsx

```diff
@@ -52,7 +52,15 @@ export const Image: React.FC<{
                     <span className={iconStyle}>
                         <TriangleIcon />
                     </span>
-                    {element.data.caption}{' '}
+
```

Comment:
LGTM, adding a TODO for separate PR. 

```suggestion
// TODO - Move caption handling to use https://github.com/guardian/dotcom-rendering/blob/master/packages/guui/components/Caption/Caption.tsx
```